packages/react/src/components/Accordion/Accordion-story.js

```diff
@@ -39,7 +39,8 @@ storiesOf('Accordion', module)
           'Accordion heading alignment (align)',
           ['start', 'end'],
           'end'
-        )}>
+        )}
+        expressive={boolean('expressive', false)}>
```

Comment:
```suggestion
        expressive={boolean('Expressive variation (expressive)', false)}>
```

to match the knob labels across the storybook